source/2019-12-06-the-ember-times-issue-127.md

```diff
@@ -15,13 +15,21 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## ["I contribute to Ember" with Ben Demboski 👨‍💻](https://discuss.emberjs.com/t/i-contribute-to-ember-with-ben-demboski/17294)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+<div class="float-right padded portrait-frame">
+  <img alt="Ben Demboski" title="Ben Demboski - Contributor to Ember" src="/images/blog/emberjstimes/ben_demboski.jpeg" />
```

Comment:
Does this file already exist or would it need to be added with this PR?

Reply:
It would need to be added. I wasn't sure where to add it.

Reply:
@rwwagner90, last time, I added an image in `source/images/blog` folder. I created a folder with publication date as the folder name to help with file organization. In the Markdown file, I referred to the image by source as `src="/images/blog/2019-10-04/juan_antonio.jpg"`. Hope that helps!

Reply:
Image added 👍